### PR TITLE
New version of the Visdom PCA Service with support to Singularity

### PR DESCRIPTION
**Pull request by andrewssobral**

### CODE REVIEW

File: PcaScripts/METADATA.json [hunk 1]
```diff
@@ -73,6 +73,24 @@
...
+			{
+				"name" : "Post_Start_Service2_Singularity",
+				"metadata" : {
+					"kind": "Script/post",
+					"commitMessage": "First commit",
+					"contentType": "text/x-groovy"
+				},
+				"file" : "resources/catalog/Post_Start_Service2_Singularity.groovy"
+			},
 			{
```
Comment (aminelouati) [inline PcaScripts/METADATA.json:94]: Could you remove 2 from Post_Start_Service2_Singularity ?

File: PcaScripts/resources/catalog/Post_Start_Service2_Singularity.groovy [hunk 2]
```diff
@@ -0,0 +1,111 @@
+/*********************************************************************************
```
Comment (mklkun) [inline PcaScripts/resources/catalog/Post_Start_Service2_Singularity.groovy:1]: Why adding new Post_Start scripts? If they are not generic, you should keep the script just in your PCA workflow, OR you should give it a significant name.

File: Visdom/METADATA.json [hunk 3]
```diff
@@ -31,24 +31,6 @@
           "contentType": "application/xml"
         },
         "file": "resources/catalog/Finish_Visdom.xml"
-      },
```
Comment (mklkun) [inline Visdom/METADATA.json:34]: Why did you delete the pause and resume states? In Docker we should handle them I think.

**Review by mklkun — APPROVED** [review]

**Review by aminelouati — APPROVED** [review]

**Review by ImenBizid — APPROVED** [review]
